### PR TITLE
Docs: made changes in command reference

### PR DESCRIPTION
**Pull request by NagaHarshita**
Moved the flags to the end of the command in the general synopsis. Made changes in the example commands of the documentation. Taken care that the example commands follow the syntax mentioned in the general synopsis.

### CODE REVIEW

File: site/reference/fn/eval/README.md [hunk 1]
```diff
@@ -101,42 +101,42 @@ fn-args:
 ```shell
 # execute container my-fn on the resources in DIR directory and
 # write output back to DIR
-$ kpt fn --image gcr.io/example.com/my-fn eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn
```
Comment (droot) [inline site/reference/fn/eval/README.md:104]: the diff is showing extra whitespace (probably) here and other places below.

File: site/reference/fn/eval/README.md [hunk 2]
```diff
@@ -101,42 +101,42 @@ fn-args:
...
 # and write output back to DIR
-$ kpt fn --image gcr.io/example.com/my-fn --network eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn --network 
 ```
 
 ```shell
 # execute container my-fn on the resource in DIR and export KUBECONFIG
 # and foo environment variable
-$ kpt fn --image gcr.io/example.com/my-fn --env KUBECONFIG -e foo=bar eval DIR
+$ kpt fn eval DIR --image gcr.io/example.com/my-fn --env KUBECONFIG -e foo=bar
```
Comment (droot) [inline site/reference/fn/eval/README.md:139]: Can you pl. run `make generate` to re-generate the Go files that `kpt fn eval --help()` uses. That will  keep `kpt fn eval --help()` in sync with these changes.

**Review by droot — CHANGES_REQUESTED** [review]

File: site/reference/fn/eval/README.md (hunk 2 above)
Comment (NagaHarshita) [reply site/reference/fn/eval/README.md]: The following commit made these changes
* Removing of the trailing spaces
* Updating the go files by `make generate`

**Review by droot — APPROVED** [review]

**Comment by frankfarzan** [comment]
Thanks for contributing! Are you follow up with making this change in the rest of the docs (e.g. in https://kpt.dev/book/)?

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**